### DISCUSSION TITLE
camera specs

### DISCUSSION THREAD

**Discussion by Zzerr0r** [microsoft/AirSim#4504] [Support Q&A]
Is there any way to find out the specs of the camera used like sensor width and height as well as focal length?

**Comment by alonfaraj** [comment]
You can find the parameters and default values here under CameraDefaults. 

https://microsoft.github.io/AirSim/settings/#available-settings-and-their-defaults

**Reply by Zzerr0r** [reply]
these specs aren't listed there but i found my answer under #2396
